tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -55,6 +55,52 @@ abstract class BaseWorksheetLspSuite(scalaVersion: String)
     } yield ()
   }
 
+  test("completion-imports") {
+    for {
+      _ <- server.initialize(
+        s"""
+           |/metals.json
+           |{
+           |  "a": {}
+           |}
+           |/a/src/main/scala/foo/Main.worksheet.sc
+           |import $$dep.`com.lihaoyi::scalatags:0.9.0`
```

Comment:
just curious, why double $$, when in PR description you use $dep?
Is it related to some replacements happening in tests?

Reply:
$$ is just so that we have $ in a string interpolation, since $ is a special character.